app/src/main/res/layout/ticket_analytics.xml

```diff
@@ -140,12 +216,12 @@
             <FrameLayout
                 android:layout_width="match_parent"
                 android:layout_height="1dp"
-                android:background="@color/color_shadow"/>
+                android:background="@color/color_shadow" />
 
             <FrameLayout
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/spacing_small"
-                android:background="@color/color_bottom_surface"/>
+                android:background="@color/color_bottom_surface" />
```

Comment:
Unneeded change.

Reply:
These changes are because of the indentation by android studio when we select the code and use ctrl + alt + shift + l.

Reply:
Yes, this is why you should not format the code in issue PR. This makes unnecessary changes not related to issue being fixed